style.css

```diff
@@ -0,0 +1,67 @@
+#main {
+    margin: 30px 8px;
+    padding: 15px;
+    border-radius: 5px;
+    background: #eee;
+}
+h1 {
+    font-size: 4rem;
+    margin-bottom: 0;
+}
+p {
+    display: block;
+    margin-block-start: 1em;
+    margin-block-end: 1em;
+    margin-inline-start: 0px;
+    margin-inline-end: 0px;
+}
+#img-div {
+    background: white;
+    padding: 10px;
+    margin: 0;
+}
+figcaption {
+    text-align: center;
+    font-size: 0.9rem;
+    /* width: 95%; */
+}
+section {
+    display: block;
+}
+body {
+    font-family: -apple-system, BlinkMacSystemFont, 'Segoe UI', 'Roboto', 'Helvetica Neue', Arial, sans-serif;
+    font-size: 1.4rem;
+    line-height: 1.5;
+    text-align: center;
+    color: #333;
+    margin: 0;
+}ul {
+    max-width: 550px;
+    margin: 0 auto 50px auto;
+    text-align: left;
+    line-height: 1.6;}
+    li {
+        display: list-item;
+        text-align: -webkit-match-parent;
+    }
+    strong {
+        font-weight: bold;
+    }a:visited {
+        color: #74638f;
+    }
+    blockquote {
+        font-style: italic;
+        max-width: 545px;
+        margin: 0 auto 50px auto;
+        text-align: left;
+    }
+    #tribute-link{ 
+         margin: 0px 50px;
+         text-align: center;
+         font-size: 1.4rem;
+         font-weight: bold;
+                           
+}
+
+    
+    
```

Comment:
you only need one extra line after you finished a block ,this is a liter error as well , please try to follow the linter rules as much as you can. 😄